tests/js/utils.js

```diff
@@ -428,6 +428,29 @@ export const unsubscribeFromAll = () => {
 	}
 };
 
+/**
+ * Creates a function that allows extra time for registry updates to have completed.
+ *
+ * @since n.e.x.t
+ *
+ * @param {Object} registry WP data registry instance.
+ * @return {Function} Function to await all registry updates since creation.
+ */
+export const createWaitForRegistry = ( registry ) => {
```

Comment:
This ended up being unnecessary, but I think it's still worth keeping as this will likely come up again as the underlying problem of registry changes after render is still a thing.